specs/jsonschema-validation.md

```diff
@@ -88,16 +88,27 @@ specification.
 ## Meta-Schema {#meta-schema}
 
 The current IRI for the default JSON Schema dialect meta-schema is
-`https://json-schema.org/draft/next/schema`. For schema author convenience, this
-meta-schema describes a dialect consisting of all keywords defined in this
+`https://json-schema.org/1/2025`. This IRI encodes the specifications version
+and release year. Because all schemas written to conform to a given version are
+guaranteed to be compatible with later releases within the same iteration, the
+meta-schema IRI `https://json-schema.org/1` is also recognized to represent the
```

Comment:
- what is "indicated iteration"?
- using a URI that can change meaning depending on the current state of the spec feels really dodgy to me -- just like "https://json-schema.org/latest" would be, because the mapping of this value can change at any time, unexpectedly, which can be disruptive to both schema authors and implementation authors.

Reply:
> using a URI that can change meaning depending on the current state of the spec feels really dodgy to me -- just like "https://json-schema.org/latest" would be, because the mapping of this value can change at any time, unexpectedly, which can be disruptive to both schema authors and implementation authors.

That's why we stated the compatibility guarantee.  Any `https://json-schema.org/1/x` schema is compatible with all `https://json-schema.org/1/x+i` specifications.  As long as the version (`1` in this case) is the same, that compatibility guarantee holds.  Because of this, it's safe to have `https://json-schema.org/1` as a `$schema` value.  Future releases within the same version will not break schemas.